docs/usecases/remodnav.rst

```diff
@@ -291,45 +290,137 @@ to fill tables with unique names with minimal work, for example like this:
 
    code/mk_figuresnstats.py -s | tee results_def.tex
 
-One can include this file as an input source into the ``.tex`` file with
+This will redirect every print statement the script wrote to the terminal into a file called
+``results_def.tex``. This file will hence be full of ``\newcommand`` definitions that contain
+the results of the computations.
+
+For step 3, one can include this file as an input source into the ``.tex`` file with
 
 .. code-block:: tex
 
    \begin{document}
    \input{results_def.tex}
 
-Upon compilation of the ``.tex`` file into a PDF, the results of the computations captured with
-``\newcommand`` definitions are inserted into the respective part of the manuscript.
+Upon compilation of the ``.tex`` file into a PDF, the results of the
+computations captured with ``\newcommand`` definitions are inserted into the respective part
+of the manuscript.
 
+The last step is to automate this procedure. So far, the script would need to be executed
+with a command line call, and the PDF compilation would require another commandline call.
 To automate this process, `Makefiles <https://en.wikipedia.org/wiki/Make_(software)>`_ can help.
+Here is the one used for the manuscript:
+
+.. code-block:: make
+
+   all: main.pdf
+
+   main.pdf: main.tex tools.bib EyeGaze.bib results_def.tex figures
+       latexmk -pdf -g $<
+
+   results_def.tex: code/mk_figuresnstats.py
+       bash -c 'set -o pipefail; code/mk_figuresnstats.py -s | tee results_def.tex'
+
+   figures: figures-stamp
+
+   figures-stamp: code/mk_figuresnstats.py
+       code/mk_figuresnstats.py -f -r -m
+       $(MAKE) -C img
+       touch $@
+
+   clean:
+       rm -f main.bbl main.aux main.blg main.log main.out main.pdf main.tdo main.fls main.fdb_latexmk example.eps img/*eps-converted-to.pdf texput.log results_def.tex figures-stamp
+       $(MAKE) -C img clean
+
+One can read a Makefile as a recipe:
+
+- Line 1: "The overall target should be ``main.pdf`` (the final PDF of
+  the manuscript)."
+
+- Line 2-3: "To make the target ``main.pdf``, the following files are required:
+  ``main.tex`` (the ``.tex`` file), ``tools.bib`` & ``EyeGaze.bib`` (bibliography files), ``results_def.tex``
+  (the results definitions), and figures (a section not covered here, about rendering figures
+  with inkscape prior to including them in the manuscript). If all of these files are present,
+  the target ``main.pdf`` can be made by running the command ``latexmk -pdf -g``"
+
+- Line 5-6: "To make the target ``results_def.tex``, the script ``code/mk_figuresnstats.py`` is
+  required. If the file is present, the target ``results_def.tex`` can be made by running the
+  command ``bash -c 'set -o pipefail; code/mk_figuresnstats.py -s | tee results_def.tex'``"
+
+This triggers the execution of the script, collection of results in ``results_def.tex``, and PDF
+compilation upon typing ``make``.
+The last three lines define that a ``make clean`` removes all computed files, and also all
+images.
```

Comment:
Please mention `datalad run make` in the conclusions. Only this final piece will fully connect the pieces and unambiguously document who the results came to be in the dataset history.